### DISCUSSION TITLE
Skip "review order" step

### DISCUSSION THREAD

**Discussion by CVZ-es** [pretix/pretix#4260] [Ideas]
Hi, 

First of all, let me thank you for all the work done.

Most of our events are free, and I'd like to know if a function to skip the "review step" could be implemented?
When it's a paid event, it's convenient to review everything is ok, but for a free event, it would be useful to save one click!

Maybe I didn't find the option to do it?

Otherwise, it would be useful.
Best
Damien

**Comment by raphaelm** [comment]
No, this is not something we intend to change
